projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/write-query-management-field.component.spec.ts

```diff
@@ -50,5 +50,15 @@ describe('WriteQueryManagementFieldComponent', () => {
   it('should create component', () => {
     expect(component).toBeTruthy();
   });
+
+  it('showResponseForm - should not show summary', () => {
+    component.showResponseForm()
+    expect(component.showSummary).toBeFalsy();
+  });
+
+  it('confirmDetails - should show summary', () => {
+    component.confirmDetails()
```

Comment:
@DavidJayakumar - missing semicolon

Reply:
This code block is moved to webapp QueryManagmentContainerComponent